### DISCUSSION TITLE
ObjectId to Date

### DISCUSSION THREAD

**Discussion by alctny** [he3-app/he3-feedback#292] [Ideas]
希望能够添加可以从 MongoDB 的 ObjectId 中分离出时间信息，比如转化成 timestamp 或者 date 之类的其他人类可读时间格式

可参考对象：
[MongoDB ObjectId ↔ Timestamp Converter](https://steveridout.com/mongo-object-time/)

**Comment by lyzhang1999** [comment]
感谢反馈，已创建 issues 跟进：https://github.com/he3-app/he3-feedback/issues/293

**Comment by zhenweiwang1990** [comment]
https://t.he3app.com/2jhx
